corpus/events/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+from config.models import Society
+from accounts.models import ExecutiveMember
+
+# Create your models here.
+class Event(models.Model):
+    CATEGORIES = [
+        ("KSS", "KSS"),
+        ("COMPETITION", "Competition"),
+        ("EXPERT_LECTURE", "Expert Lecture"),
+        ("WORKSHOP", "Workshop"),
+        ("HACKATHON", "Hackathon"),
+    ]
+
+    VISIBILITIES = [
+        ("INTERNAL", "Internal"),
+        ("EXTERNAL", "External"),
+    ]
+    title = models.CharField(max_length=128, blank=False)
+    description = models.TextField(null=False)
+    start_time = models.DateTimeField(null=False)
+    end_time = models.DateTimeField(null=False)
+    registration_url = models.URLField(null=False, unique=True)
+    meeting_url = models.URLField(blank=True)
+    instagram_url = models.URLField(blank=True)
+    linkedin_url = models.URLField(blank=True)
+    location = models.CharField(null=False, max_length=32)
+    category = models.CharField(null=False, choices=CATEGORIES)
+    society = models.ForeignKey(Society, on_delete=models.CASCADE)
```

Comment:
What about inter-SIG events? Do we mark one SIG to be primary or something?

Reply:
Not sure how we would handle that. Should I have this be a ManyToMany field? Each society can have many events and an event can have many societies?